flink-catalog-examples/README.md

```diff
@@ -0,0 +1,273 @@
+```
+                                   ▒▓██▓██▒
+                               ▓████▒▒█▓▒▓███▓▒
+                            ▓███▓░░        ▒▒▒▓██▒  ▒
+                          ░██▒   ▒▒▓▓█▓▓▒░      ▒████
+                          ██▒         ░▒▓███▒    ▒█▒█▒
+                            ░▓█            ███   ▓░▒██
+                              ▓█       ▒▒▒▒▒▓██▓░▒░▓▓█
+                            █░ █   ▒▒░       ███▓▓█ ▒█▒▒▒
+                            ████░   ▒▓█▓      ██▒▒▒ ▓███▒
+                         ░▒█▓▓██       ▓█▒    ▓█▒▓██▓ ░█░
+                   ▓░▒▓████▒ ██         ▒█    █▓░▒█▒░▒█▒
+                  ███▓░██▓  ▓█           █   █▓ ▒▓█▓▓█▒
+                ░██▓  ░█░            █  █▒ ▒█████▓▒ ██▓░▒
+               ███░ ░ █░          ▓ ░█ █████▒░░    ░█░▓  ▓░
+              ██▓█ ▒▒▓▒          ▓███████▓░       ▒█▒ ▒▓ ▓██▓
+           ▒██▓ ▓█ █▓█       ░▒█████▓▓▒░         ██▒▒  █ ▒  ▓█▒
+           ▓█▓  ▓█ ██▓ ░▓▓▓▓▓▓▓▒              ▒██▓           ░█▒
+           ▓█    █ ▓███▓▒░              ░▓▓▓███▓          ░▒░ ▓█
+           ██▓    ██▒    ░▒▓▓███▓▓▓▓▓██████▓▒            ▓███  █
+          ▓███▒ ███   ░▓▓▒░░   ░▓████▓░                  ░▒▓▒  █▓
+          █▓▒▒▓▓██  ░▒▒░░░▒▒▒▒▓██▓░                            █▓
+          ██ ▓░▒█   ▓▓▓▓▒░░  ▒█▓       ▒▓▓██▓    ▓▒          ▒▒▓
+          ▓█▓ ▓▒█  █▓░  ░▒▓▓██▒            ░▓█▒   ▒▒▒░▒▒▓█████▒
+           ██░ ▓█▒█▒  ▒▓▓▒  ▓█                █░      ░░░░   ░█▒
+           ▓█   ▒█▓   ░     █░                ▒█              █▓
+            █▓   ██         █░                 ▓▓        ▒█▓▓▓▒█░
+             █▓ ░▓██░       ▓▒                  ▓█▓▒░░░▒▓█░    ▒█
+              ██   ▓█▓░      ▒                    ░▒█▒██▒      ▓▓
+               ▓█▒   ▒█▓▒░                         ▒▒ █▒█▓▒▒░░▒██
+                ░██▒    ▒▓▓▒                     ▓██▓▒█▒ ░▓▓▓▓▒█▓
+                  ░▓██▒                          ▓░  ▒█▓█  ░░▒▒▒
+                      ▒▓▓▓▓▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░▓▓  ▓░▒█░
+
+    ______ _ _       _       _____  ____  _         _____ _ _            _  BETA   
+   |  ____| (_)     | |     / ____|/ __ \| |       / ____| (_)          | |  
+   | |__  | |_ _ __ | | __ | (___ | |  | | |      | |    | |_  ___ _ __ | |_
+   |  __| | | | '_ \| |/ /  \___ \| |  | | |      | |    | | |/ _ \ '_ \| __|
+   | |    | | | | | |   <   ____) | |__| | |____  | |____| | |  __/ | | | |_
+   |_|    |_|_|_| |_|_|\_\ |_____/ \___\_\______|  \_____|_|_|\___|_| |_|\__|
+
+        Welcome! Enter 'HELP;' to list all available commands. 'QUIT;' to exit.
+
+
+Flink SQL>
+```
+
+After that, we need to create a Pravega Catalog to access the data stored in Pravega.
+In order to use Pravega Catalog, we need to add schema registry group and register the schema of your serialization format to schema registry service.
+The datagen container has done these things, so the catalog can get the schema information, translate the Pravega stream into Flink table and show in the catalog.
```

Comment:
This line is more about the container introduction, please consider move to the above section.